csharp_dotnetcore/11.QnAMaker/appsettings.json

```diff
@@ -0,0 +1,5 @@
+{
+  //  The MicrosoftAppId and MicrosoftAppPassword is created from the Azure portal http://portal.azure.com when you created your Bot.
+  "MicrosoftAppId": "",
```

Comment:
Should remove these from app setting and move them to the .bot file. Thus remove from this file